lib/sorbet-rails/tasks/rails_rbi.rake

```diff
@@ -3,10 +3,6 @@ require("sorbet-rails/routes_rbi_formatter")
 require("sorbet-rails/helper_rbi_formatter")
 require("sorbet-rails/utils")
 
-# this is ugly but it's a way to get the current directory of this script
-# maybe someone coming along will know a better way
-@@sorbet_rails_rake_dir = File.dirname(__FILE__)
```

Comment:
Could a simpler way to resolve this be to define an inline class with a constant?

eg:
```
class Foo
  RAKE_DIR = ...
end
```

And then reference `Foo::RAKE_DIR` at the callsite?

Reply:
I will close this PR.